### ISSUE TITLE
Vulnerabilities from tmi.js

### ISSUE THREAD

**Issue by chriteixeira** [chriteixeira/twitch-toolkit#5]
The project tmi.js have a few vulnerabilities and looks like its not being updated anymore (last npm publish was from a year ago).

A few options to solve this:
- The repository have not yet published commits. Check if its stable and fork it to be used in this project.
- Fork it and solve it (maybe create a new npm project)
- Use a new chat module (maybe https://github.com/marcandrews/twitch-js).

**Comment by bradnak** [comment]
Yeah tmi.js is dead now. I believe they recommend [twitch-js](https://github.com/twitch-js/twitch-js) now (which is what the Marc Andrews one is forked from). I was looking into using your package in my project for the PubSub module, but the vulnerabilities in tmi.js make me a little hesitant to use it.
